gpt4all-chat/qml/AddModelView.qml

```diff
@@ -187,7 +187,20 @@ Rectangle {
                 visible: false
                 MyComboBox {
                     id: comboSort
-                    model: [qsTr("Default"), qsTr("Likes"), qsTr("Downloads"), qsTr("Recent")]
+                    model: ListModel {
+                        ListElement {
+                            name: qsTr("Default")
+                        }
+                        ListElement {
+                            name: qsTr("Likes")
+                        }
+                        ListElement {
+                            name:  qsTr("Downloads")
+                        }
+                        ListElement {
+                            name:  qsTr("Recent")
+                        }
+                    }
```

Comment:
Would it be OK if we made these more compact?
```suggestion
                    model: ListModel {
                        ListElement { name: qsTr("Default")   }
                        ListElement { name: qsTr("Likes")     }
                        ListElement { name: qsTr("Downloads") }
                        ListElement { name: qsTr("Recent")    }
                    }
```

Reply:
sure